core/backend/src/NativeHost.ts

```diff
@@ -22,16 +22,23 @@ import { NativeAppStorage } from "./NativeAppStorage";
 /** @internal */

```

Comment:
do we think this would be moved to alpha/beta soon? We're going to ultimately use the solution in the desktop-starter, so we would need it to be public first

Reply:
no, this should stay internal. It's only exported so it can be subclassed by Electron and Mobile. You should not need to use it in desktop-starter, right?

Reply:
Correct. We would not use it directly.